src/commcare_cloud/ansible/setup_security_updates.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  become: yes
+  roles:
+   - security_upgrades
+  vars:
+   apt_cache_valid_time: 7200
+   apt_unattended_upgrades_blacklist: 
+    - postgresql-9.6
+    - postgresql-9.6-plproxy
+    - postgresql-client-9.6
+    - postgresql-client-9.6
+    - postgresql-contrib-9.6
+    - postgresql-server-dev-9.6
+    - rabbitmq-server
+    - riak
+    - riak-cs
+    - zookeeper
```

Comment:
I don't think we need to blacklist the postgres clients. Otherwise this looks like a good list.